src/map/script.c

```diff
@@ -10688,6 +10691,8 @@ static BUILDIN(setmount)
 	if (script_hasdata(st,2))
 		flag = script_getnum(st,2);
 
+	enum mado_type mtype = script_hasdata(st, 3) ? script_getnum(st, 3) : MADO_ROBOT;
```

Comment:
what if they pass any number other than `MADO_ROBOT` and `MADO_SUITE`? 
which the madogear types doesn't exists or undefined.

Reply:
We check for valid type in `mtype < MADO_ROBOT || mtype >= MADO_MAX` and show an error, anything else is handled by the client, for Gravity's clients they default anything that is not valid to 0 which is robo mado.